docs/contributing.rst

```diff
@@ -54,11 +54,17 @@ Then start the ``compose`` session::
 
 At this point, you should be able to see the Web UI at http://localhost:8000
 
-Once your session is up and running, you can make yourself a superuser account::
+Once your session is up and running, you can::
+   
```

Comment:
```suggestion
Once your session is up and running, you can:

```

Reply:
I noticed one warning in the build log: https://github.com/jointakahe/takahe/actions/runs/3613790002/jobs/6089780689#step:5:28

This is due to the double colon syntax `::` called literal blocks. Only we need to put two colons when placing a code block in the next paragraph. So you don't have to place two here. 🙂 

ref. [reStructuredText Primer — Sphinx documentation](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks)

Reply:
You can build documentation with the following commands:

```shell
cd takahe/
pip install -r docs/requirements.txt
make docs
```

This outputs HTML files under the `docs/_build/html/` directory.

Reply:
Looks good! 👍🏻